overrides/config/deepmobevolution/DataModelTiers.json

```diff
@@ -0,0 +1,56 @@
+[
+  {
+    "tier": 0,
+    "displayName": "Basic",
+    "displayColor": "green",
+    "killMultiplier": 4,
+    "dataToNext": 48,
```

Comment:
At a glance, I can tell that the data model stats differ from the ones we have been using.